tests/core/dom/range/manual/getclientrects.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: 4.10.0, feature, range, selection
+
+# Selection rectangles
+
+This manual test checks the result returned by the `range.getClientRects()` method. Returned results are visualized with red rectangles.
+
+## Things to test:
+
+Select some part of the editable and see if red rectangles match the selection. Play around with the selection using mouse and keyboard (`shift + arrow keys`).
+
+### Note:
```

Comment:
Inconsistent spacing, above you added a line after header, here not. Just add a single line for readability.